### PR TITLE
Feature/start building explore

### PR DESCRIPTION
**Pull request by edwardgaudio**
**Is your addition related to a problem? Please describe.**
Make start building and explore sections style consistent with work done previously for stateless contracts, up to par with modern js. Also I ran across some oddities I just ended up removing or adjusting.

**Describe the solution you'd like**
Same as before, combination of main async function with then/catch promise syntax.

**Additional context**
WIP at the moment unless we wanna break this up into multiple pr's

### CODE REVIEW

File: docs/features/transactions/signatures.md [hunk 1]
```diff
@@ -131,73 +131,24 @@ Extend the example from the [Multisignature Account](../accounts/create.md#multi
...
-                    throw new Error("Transaction Rejected" + " pool error" + pendingInfo["pool-error"]);
-                }
-            }
-        }
-        await algodclient.statusAfterBlock(currentround).do();
-        currentround++;
-    }
-    throw new Error("Transaction not confirmed after " + timeout + " rounds!");
+const waitForConfirmation = async function (algodClient, txId, timeout) {
+    // ...covered in a previous example
```
Comment (ryanRfox) [inline docs/features/transactions/signatures.md:135]: I feel this method must be included within this script as the reader may not access related content and we want the code to be complete inline.

File: examples/accounts/v2/javascript/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (ryanRfox) [inline examples/accounts/v2/javascript/package.json:1]: These package.json files should be removed, as the license is covered here:
https://github.com/algorand/docs/blob/staging/LICENSE.md

File: examples/multisig/v2/javascript/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (ryanRfox) [inline examples/multisig/v2/javascript/package.json:1]: remove

File: examples/smart_contracts/v1/javascript/AccountDelegation.js [hunk 4]
```diff
@@ -1,76 +1,56 @@
...
-    let rawSignedTxn = algosdk.signLogicSigTransaction(txn, lsig);
-
-    // send raw LogicSigTransaction to network  
-    console.log("This is expected to fail as the program is int 0 and will return false : ");      
-    let tx = (await algodclient.sendRawTransaction(rawSignedTxn.blob));
+    // *** End account delegation changes ***
+
+    // Setup a transaction
+    const sender = myAccount.addr;
+    const receiver = "HZ57J3K46JIJXILONBBZOHX6BKPXEM2VVXNRFSUED6DKFD5ZD24PMJ3MVA";
```
Comment (ryanRfox) [inline examples/smart_contracts/v1/javascript/AccountDelegation.js:41]: Please us v2 dispenser HZ57J3K46JIJXILONBBZOHX6BKPXEM2VVXNRFSUED6DKFD5ZD24PMJ3MVA

File: examples/smart_contracts/v1/javascript/ContractAccount.js [hunk 5]
```diff
@@ -1,69 +1,64 @@
...
-    };
-    // Create the LogicSigTransaction with contract account LogicSig 
-    let rawSignedTxn = algosdk.signLogicSigTransaction(txn, lsig);
+    const lsig = algosdk.makeLogicSig(program, args);
+    console.log("lsig : " + lsig.address());   
+
+    // Create a transaction
+    const sender = lsig.address();
+    // This is a sandbox address and will be different in your environment
+    const receiver = "GVVIH6IE3ZAFLIQF6UFBOH4TI5ZOPHFRMDQUTXQRKZ6XWSRTVRDBQ4GZLY";
```
Comment (ryanRfox) [inline examples/smart_contracts/v1/javascript/ContractAccount.js:47]: HZ57J3K46JIJXILONBBZOHX6BKPXEM2VVXNRFSUED6DKFD5ZD24PMJ3MVA

File: examples/smart_contracts/v1/javascript/package.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (ryanRfox) [inline examples/smart_contracts/v1/javascript/package.json:1]: remove

File: examples/smart_contracts/v1/javascript/utils/waitForConfirmation.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+// Function used to wait for a tx confirmation
```
Comment (ryanRfox) [inline examples/smart_contracts/v1/javascript/utils/waitForConfirmation.js:1]: I like this :)

File: examples/smart_contracts/v2/javascript/package.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (ryanRfox) [inline examples/smart_contracts/v2/javascript/package.json:1]: remove

**Review by ryanRfox — CHANGES_REQUESTED** [review]
Good stuff here. Just a few minor changes.

**Comment by ryanRfox** [comment]
ping @edwardgaudio just a couple edit to get us over the line :)

**Review by ryanRfox — APPROVED** [review]
LGTM

**Comment by edwardgaudio** [comment]
> LGTM

Conflicts resolved and ready to go @ryanRfox

**Pull request closed without merging**